docs/pages/configuration/images/build-engines/custom.mdx

```diff
@@ -51,5 +51,5 @@ images:
       - some/path/
       - another/path/file.txt
 ```
-**Explanation:**  
+**Explanation:**
 The image `backend` would be built using the command `./build --image "[IMAGE]:[TAG]"` and DevSpace would skip the build if none of the files within `some/path/` nor the file `another/path/file.txt` has changed since the last build.
```

Comment:
```suggestion
The image `backend` is built using the command `./build --image "[IMAGE]:[TAG]"`, and DevSpace skips the build if none of the files within `some/path/`, nor the file `another/path/file.txt`, has changed since the last build.
```